lightwood/analysis/helpers/acc_stats.py

```diff
@@ -21,7 +21,7 @@ def analyze(self, info: Dict[str, object], **kwargs) -> Dict[str, object]:
 
         # @TODO: maybe pass ts_analysis to trigger group-wise MASE instead of R2 mean, though it wouldn't be 0-1 bounded
         info['score_dict'] = evaluate_accuracy(ns.data, ns.normal_predictions['prediction'],
-                                               ns.target, ns.accuracy_functions)
+                                               ns.target, ns.accuracy_functions, ts_analysis={'tss': ns.ts_cfg})
```

Comment:
small thing, but ahm, why call it `ts_cfg` and not `tss` or `timeseries_settings`, kinda confusing to use different names

Reply:
That naming scheme is not part of this PR, but I agree it's confusing. I'll change it to `tss` for consistency.